db/migrate/20201116182332_create_bookings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBookings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookings do |t|
+      t.string :start_date
+      t.string :days
```

Comment:
I think `days` should be an integer as it represents the number of days that the item is loaned for (which will always be a whole number)

Reply:
This has been changed. Check to confirm this.